ci/matrix.xml

```diff
@@ -174,7 +174,8 @@
   <task id="tools-cli-stubs"><![CDATA[
     setJavaVersion $java
     sbt ++$scala tools/package ir/test tools/test cli/package cli/assembly \
-        stubs/package jsEnvsTestSuite/test \
+        stubs/package jsEnvsTestSuite/test phantomJSEnv/test \
```

Comment:
Duh, yes, I wanted to make a separate commit for that. I forgot when I made the commit, I guess.